### PR TITLE
osfv_cli: correct the flashing power state logic

### PR DESCRIPTION
**Pull request by miczyg1**
Use ACPI nomenclature to be consistent and rename OFF to S5. Both OFF and G3 mean powered off state, however ACPI differentiates soft off with PSU/AC active (S5) and hard off with PSU/AC inactive (G3).

Also the logic controlling the power state during flashing was inverted. We always put the device to S5 state (PSU/AC active) before attempting to flash. But for G3 path, the PSU/AC stayed on, while should be turned off. All board configs have been changed to their proper states to reflect the correct power logic.

### CODE REVIEW

File: osfv_cli/src/osfv/libs/rte.py [hunk 1]
```diff
@@ -193,20 +193,23 @@ def pwr_ctrl_before_flash(self, programmer, power_state):
         # Always start from the same state (PSU active)
```
Comment (krystian-hebel) [inline osfv_cli/src/osfv/libs/rte.py:193]: Please update also line 55 of this file, is still has OFF in it.

Comment (miczyg1) [reply osfv_cli/src/osfv/libs/rte.py:193]: https://github.com/Dasharo/osfv-scripts/compare/6746f61422851415a6b3cca2c1f26e9d177fa288..ab7e35b33e7925cd8bd50eb871c91af45d6843f0

Comment (miczyg1) [reply osfv_cli/src/osfv/libs/rte.py:193]: Thanks, forgot about it

**Comment by macpijan** [comment]
README change was missing. Pushed it to this branch, feel free to suqash if you prefer.

**Review by krystian-hebel — APPROVED** [review]

**Review by macpijan — APPROVED** [review]

**Pull request merged**